src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/BaseIntegrationTest.java

```diff
@@ -27,6 +27,7 @@
 
     protected static final String SUBMITTER_ID = "123";
     protected static final String DEFENDANT_ID = "555";
+    public static final String BEARER_TOKEN = "Bearer let me in";
```

Comment:
Please make it protected to discourage anyone from reusing it outside tests that inherit the base class.